systemtest/src/test/java/io/strimzi/systemtest/operators/MultipleClusterOperatorsST.java

```diff
@@ -0,0 +1,247 @@
+import io.strimzi.systemtest.resources.crd.kafkaclients.KafkaBasicExampleClients;
+import io.strimzi.systemtest.resources.operator.BundleResource;
+import io.strimzi.systemtest.utils.ClientUtils;
+import io.strimzi.systemtest.utils.kafkaUtils.KafkaConnectUtils;
+import io.strimzi.systemtest.utils.kafkaUtils.KafkaRebalanceUtils;
+import io.strimzi.systemtest.utils.kafkaUtils.KafkaUtils;
+import io.strimzi.systemtest.utils.kubeUtils.controllers.DeploymentUtils;
+import io.strimzi.systemtest.utils.kubeUtils.controllers.StatefulSetUtils;
+import io.strimzi.systemtest.utils.kubeUtils.objects.PodUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import static org.hamcrest.CoreMatchers.is;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static io.strimzi.systemtest.Constants.REGRESSION;
+import static io.strimzi.test.k8s.KubeClusterResource.kubeClient;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
+
+@Tag(REGRESSION)
+public class MultipleClusterOperatorsST extends AbstractST {
+
+    private static final Logger LOGGER = LogManager.getLogger(MultipleClusterOperatorsST.class);
+
+    public static final String DEFAULT_NAMESPACE = "multiple-co-cluster-test";
+    public static final String FIRST_NAMESPACE = "first-co-namespace";
+    public static final String SECOND_NAMESPACE = "second-co-namespace";
+
+    public static final String FIRST_CO_NAME = "first-" + Constants.STRIMZI_DEPLOYMENT_NAME;
+    public static final String SECOND_CO_NAME = "second-" + Constants.STRIMZI_DEPLOYMENT_NAME;
+
+    public static final EnvVar FIRST_CO_SELECTOR_ENV = new EnvVar("STRIMZI_CUSTOM_RESOURCE_SELECTOR", "co-name=" + FIRST_CO_NAME, null);
+    public static final EnvVar SECOND_CO_SELECTOR_ENV = new EnvVar("STRIMZI_CUSTOM_RESOURCE_SELECTOR", "co-name=" + SECOND_CO_NAME, null);
+
+    public static final Map<String, String> FIRST_CO_SELECTOR = Collections.singletonMap("co-name", FIRST_CO_NAME);
+    public static final Map<String, String> SECOND_CO_SELECTOR = Collections.singletonMap("co-name", SECOND_CO_NAME);
+
+    @Test
+    void testMultipleCOsWithDifferentCustomResourceSelector() {
+        deployCOInNamespace(FIRST_CO_NAME, DEFAULT_NAMESPACE, FIRST_CO_SELECTOR_ENV, false);
+        deployCOInNamespace(SECOND_CO_NAME, DEFAULT_NAMESPACE, SECOND_CO_SELECTOR_ENV, false);
+
+        testMultipleCOsWithDifferentCRSelectors();
+    }
+
+    @Test
+    void testMultipleCOsInDifferentNamespaces() {
+        deployCOInNamespace(FIRST_CO_NAME, FIRST_NAMESPACE, FIRST_CO_SELECTOR_ENV, true);
+        deployCOInNamespace(SECOND_CO_NAME, SECOND_NAMESPACE, SECOND_CO_SELECTOR_ENV, true);
+
+        cluster.setNamespace(DEFAULT_NAMESPACE);
+
+        testMultipleCOsWithDifferentCRSelectors();
+    }
```

Comment:
I'm not completel sure what is the added value of these two tests compared to `testLabelingAndUnlabelingOfResource`. It looks like they test the same and just add Connect and Connector. Maybe these tests should be joined? Either keep these and add the stuff from `testLabelingAndUnlabelingOfResource` here or the other way around. Tat should save some execution time.

I'm also not sure we really need to test separately both `testMultipleCOsInDifferentNamespaces` and `testMultipleCOsWithDifferentCustomResourceSelector`. I do not think that after it is installed the operator cares whether there are two of them running in the same or in different namespace.